packages/edit-post/src/components/header/admin-menu-toggle/index.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { Button } from '@wordpress/components';
+import { useRef, useEffect, useState } from '@wordpress/element';
+import { ESCAPE, TAB } from '@wordpress/keycodes';
+import { __ } from '@wordpress/i18n';
+import { wordpress } from '@wordpress/icons';
+
+function AdminMenuToggle() {
```

Comment:
Since we'll need the same component in Site Editor and maybe Widgets screen, could we move it to `@wordpress/interface` packaged so it can be more easily reused in that context?

Reply:
Cool, I managed to figure out how to move it to the interface package. Should be good to go now.